<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_RGB" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_G">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>47</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_B">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>77</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_B">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_G">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_B">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_R">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>17</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_R">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_R">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_G">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clearColorButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Faz a cor selecionada voltar para preto (ou seja, (0, 0, 0))</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>Limpar cor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendColorButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Faz a cor selecionada voltar para preto (ou seja, (0, 0, 0))</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>Enviar cor</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_R</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_R</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_R</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_R</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_G</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_G</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_G</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_G</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_B</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_B</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_B</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_B</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
